Content.Shared/Examine/ExamineSystemShared.cs

```diff
@@ -194,7 +202,7 @@ public bool InRangeUnOccluded(MapCoordinates origin, MapCoordinates other, float
                 }
 
                 var bBox = o.BoundingBox;
-                bBox = bBox.Translated(entMan.GetComponent<TransformComponent>(result.HitEntity).WorldPosition);
```

Comment:
all changes in this file after here can be kept

Reply:
Well done